src/lib/containers/UserCardSmall.tsx

```diff
@@ -1,92 +1,102 @@
+  const [show, setShow] = useState(false)
 
-  const imgClassName = `SRC-userImgSmall ${extraSmall ? 'extraSmall' : ''} ${
-    preSignedURL ? '' : 'SRC-centerContentInline'
-  } `
-  if (preSignedURL) {
-    marginLeft = '3px'
-    img = (
-      <div
-        className={imgClassName}
-        style={{ backgroundImage: `url(${preSignedURL})` }}
-        data-for={label}
-        data-tip={label}
-      />
-    )
-  } else {
-    const color = getColor(userProfile.userName)
-    marginLeft = '3px'
-    img = (
-      <div style={{ background: color }} className={imgClassName}>
-        {(userProfile.firstName && userProfile.firstName[0]) ||
-          userProfile.userName[0]}
-      </div>
-    )
-  }
-  return (
+  const target = useRef(null)
+
+  const OverlayComponent = (
+    <Overlay target={target.current} show={show} placement="top-start">
+      {({ placement, arrowProps, show: _show, popper, ...props }) => {
+        return (
+          <div
+            className="bootstrap-4-backport"
+            onMouseEnter={() => {
+              resetTimer(timer)
+              setShow(true)
+            }}
+            onMouseLeave={() => {
+              // Assume the user is done with the card regardless of click state
+              resetTimer(timer)
+              timer.current = setTimeout(() => {
+                setShow(false)
+              }, TIMER_DELAY_HIDE)
+            }}
+            {...props}
+            style={{ ...props.style, width: 'max-content', minWidth: '300px' }}
+          >
+            <UserCard
+              size={MEDIUM_USER_CARD}
+              ownerId={userProfile.ownerId}
+              link={link}
+              disableLink={disableLink}
+              {...rest}
+            />
+          </div>
+        )
+      }}
+    </Overlay>
+  )
+
+  return showCardOnHover ? (
+    <>
+      {OverlayComponent}
+      <span
+        ref={target}
+        onMouseEnter={() => {
+          resetTimer(timer)
+          timer.current = setTimeout(() => setShow(true), TIMER_DELAY_SHOW)
+        }}
+        onMouseLeave={() => {
+          resetTimer(timer)
+          timer.current = setTimeout(() => setShow(false), TIMER_DELAY_HIDE)
+        }}
+        onClick={() => {
+          resetTimer(timer)
+          setShow(!show)
+        }}
+        className="SRC-userCard UserCardSmall SRC-primary-text-color SRC-underline-on-hover"
+        style={{ whiteSpace: 'nowrap' }}
+      >{`@${userProfile.userName}`}</span>
+    </>
+  ) : disableLink ? (
+    <span
+      className="SRC-userCard UserCardSmall SRC-boldText"
+      style={{ cursor: 'unset' }}
+    >{`@${userProfile.userName}`}</span>
+  ) : (
     <a
-      href={linkLocation}
-      className={
-        'SRC-userCard UserCardSmall SRC-primary-text-color SRC-no-underline-on-hover '
+      className="SRC-userCard UserCardSmall"
+      href={
+        link ? link : `https://www.synapse.org/#!Profile:${userProfile.ownerId}`
       }
-    >
-      {img}
```

Comment:
Are you removing all profile images from small user cards because I need a version of the small user card that shows the profile image without hover. Thanks.

![Screen Shot 2021-04-09 at 2 13 39 PM](https://user-images.githubusercontent.com/434792/114241702-71ba7800-993e-11eb-96a2-948944869706.png)

Reply:
Not sure if this is true, need to confirm with Ljubo (he was pushing for this new small user card definition).  
@emmanmills : Is this for the ManagedACTAccessRequirement DataAccessRequest accessor (user) list?  If so, could you reach out to Stockard and Ljubo to get a decision on this?  I'd like to apply this small user card (with popover) across the web apps (including all small user badges on synapse.org).

Reply:
Stockard said it's okay to remove the profile image.